spec/requests/api/user_can_create_a_post_spec.rb

```diff
@@ -0,0 +1,45 @@
+RSpec.describe 'POST /api/posts', type: :request do
+  describe 'successfully create a post' do
+    before do
+      post '/api/posts',
+           params: {
+             post: {
+               track: 'One Beer',
+               artists: 'MF DOOM',
```

Comment:
change the song to "CELLZ" just for the fun of it

Reply:
We will go for this song for now, we can add the other one next time!